### DISCUSSION TITLE
Side Panel Configuration Does Not Yield a Row

### DISCUSSION THREAD

**Discussion by mmalessa80** [Budibase/budibase#10268] [Help] [answered]
I am attempting to make the newer side panel feature work, however the automatic configuration in the Details Form Block for Row ID is {{ [state].[ID_RRQ34JDn2] }} which does not actually result in the details of the clicked row being shown. The Title reflects the key value of the first row only but no data is reflected in the details. Could someone assist please?

![image](https://user-images.githubusercontent.com/106758851/231246342-4251d94b-7673-42bc-91e7-6a00a58895b8.png)

**Comment by ConorWebb96** [comment, marked as answer]
Hey @mmalessa80,

I don't think you have selected a table to match the row against? Are you sure the ID you are storing in state matches against one you are trying to fetch?

I've created an example below to show you this process from start to finish.

**Component setup**
<img width="265" alt="component breakdown" src="https://user-images.githubusercontent.com/126772285/231430055-5a8dd248-fc74-4ed7-81db-439dbea9471b.png">

**Creating the state**
<img width="1607" alt="createdState" src="https://user-images.githubusercontent.com/126772285/231430120-d9642b77-9168-4042-9b49-7f741e9b2732.png">

**Sidepanels breakdown - table or query attachment is needed**
<img width="738" alt="sidepanel breakdown" src="https://user-images.githubusercontent.com/126772285/231430306-49ce380c-8cb1-4691-ab20-55b2f9bf6b28.png">

**State being applied on the row**
<img width="1604" alt="state applied to row" src="https://user-images.githubusercontent.com/126772285/231430385-8c19909b-4b48-4a81-9f5f-1c3f3999d9b8.png">

I hope this helps.

**Reply by mmalessa80** [reply]
Thank you! I think the main issue I was having is that the documentation on the side panel feature does not contain any information on the 'Update State' action, unless I am looking in the wrong place. Here is a link to the documentation I was using https://docs.budibase.com/docs/side-panel. And since in my case I used the eject block from the table block as is documented under Table blocks it automatically assigned a binding to the side panel form block, I interpreted that this would just work out of the box, rather than having to configure additional items. For clarification purposes this is the documentation on the Table block I am referring to https://docs.budibase.com/docs/table-block. I am unsure whom to contact or flag to maybe update the documentation?

**Reply by ConorWebb96** [reply]
Thanks for bringing this to our attention, I've made a note. We will have a look at the documentation you've mentioned and try to improve them in order to avoid similar confusion in the future!

Thanks again for the feedback!
